### PR TITLE
optional output for setadetails

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

File: src/server.py [hunk 1]
```diff
@@ -468,11 +479,55 @@ def run_seta_details_query():
...
         parts = d[0].split("'")
         jobtype.append([parts[1], parts[3], parts[5]])
-    retVal[date] = jobtype
 
+    if active:
+        alljobs = jobtype_query()
+        active_jobs = []
+        for job in alljobs:
+            found = False
+            for j in jobtype:
```
Comment (dminor) [inline src/server.py]: isn't jobtype always an empty list at this point?

Comment (jmaher) [inline src/server.py]: jobtype is the array of jobtypes from the query (jobs we can run optionally), alljobs is the master list of all jobtypes, so here we are building a list of active_jobs which are alljobs-jobtype

Comment (dminor) [inline src/server.py]: Oh ok, I see it now

**Comment by dminor** [comment]
Looks good to me

**Pull request merged**